pallets/nft/src/lib.rs

```diff
@@ -461,35 +474,39 @@ pub mod pallet {
         pub fn claim(origin: OriginFor<T>, nft: NftOf<T>) -> DispatchResult {
```

Comment:
We might need some comments explaining claim rules. It's a large function with lots of contexts.